### PR TITLE
Create 78.html

### PR DESCRIPTION
**Pull request by Yolgie**

### CODE REVIEW

**Comment by Yolgie** [comment]
not sure what the old map coordinates are or if we can just omit them

used the embed link for google maps, or should this be a different one?

also did not see the location image used in the current techplauscherl html page so not sure what the format should be :)

**Comment by Yolgie** [comment]
@hupfis is this correct like this :)

File: _plauscherl/78.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: "Technologieplauscherl LXXVIII"
+permalink: /78/
+name: 78
+layout: post
+datestring: "30. November 2022"
+timestring: "18:30"
+startDateIso8601: "2022-11-30T18:30"
+event:
+  meetup:
+  facebook:
```
Comment (hupfis) [inline _plauscherl/78.html:11]: 441756534713264

File: _plauscherl/78.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  linkedin:
```
Comment (hupfis) [inline _plauscherl/78.html:12]: 6986039174076219393

**Review by hupfis — CHANGES_REQUESTED** [review]
Please add the event ids of the various platforms

**Pull request merged**